app/BUILD.bazel

```diff
@@ -804,6 +806,7 @@ crashlytics_android_library(
 android_library(
     name = "crashlytics_deps",
     exports = [
+        "//third_party:androidx_core_core",
```

Comment:
Per my other comment, please remove this & the one from binding_adapters, and add it to view_models.

Reply:
Done.